playbooks/roles/lets-encrypt/tasks/main.yml

```diff
@@ -0,0 +1,58 @@
+- name: Add the APT key for acmetool
+  apt_key:
+    id: 9862409EF124EC763B84972FF5AC9651EDB58DFA
+    data: "{{ item }}"
+  with_file: acmetool_ppa.pem
+
+- name: Add the official acmetool repository
+  apt_repository:
+    repo: "deb http://ppa.launchpad.net/hlandau/rhea/{{ ansible_distribution|lower }} {{ ansible_lsb.codename }} main"
+
+- name: Install acmetool
```

Comment:
Can you put the first three tasks (adding the apt key, adding the repository, installing the apt package) into their own `install.yml` task file that's included here? Thanks!